consul/consul_test.go

```diff
@@ -1057,3 +1057,51 @@ func Test_substituteEnvironment(t *testing.T) {
 		})
 	}
 }
+
+func TestACLTokenPassing(t *testing.T) {
```

Comment:
Change name of this tests to define what happens here. e.g., `TestIfTokenIsUsedToConnectToConsul`

Reply:
Changed to TestTokenIsUsedToConnectToConsul